chart/openfaas-oidc-proxy/README.md

```diff
@@ -74,59 +73,19 @@ EOF
 kubectl apply -f issuer-prod.yaml
 ```
 
-Create an ingress record for the openfaas-oidc-proxy:
-
-```bash
-export DOMAIN="gateway.example.com"
-
-cat > ingress.yaml <<EOF
-apiVersion: networking.k8s.io/v1
-kind: Ingress
-metadata:
-  name: openfaas-oidc-proxy
-  namespace: openfaas
-  annotations:
-    kubernetes.io/ingress.class: nginx
-    cert-manager.io/issuer: letsencrypt-prod
-  labels:
-    app: openfaas-oidc-proxy
-spec:
-  tls:
-  - hosts:
-    - $DOMAIN
-    secretName: openfaas-oidc-proxy-cert
-  rules:
-  - host: $DOMAIN
-    http:
-      paths:
-      - path: /
-        pathType: Prefix
-        backend:
-          service:
-            name: openfaas-oidc-proxy
-            port:
-              number: 8080
-EOF
-```
-
-Apply the Ingress resource:
-
-```bash
-kubectl apply -f ingress.yaml
-```
-
 ## Configure values.yaml
 
 ```yaml
-# OpenFaaS gateway URL
-gatewayURL: http://gateway.openfaas:8080
-
 # The public URL to access the proxy
-publicURL: gateway.example.com
+publicURL: oidc-proxy.example.com
 
 # Comma separated list of repository owners for which short-lived OIDC tokens are authorized.
-# For example: openfaasltd,self-actuated
-repositoryOwners: 'openfaasltd,self-actuated'
+# For example: alexellis,self-actuated
+repositoryOwners: 'alexellis,self-actuated'
+
+ingress:
```

Comment:
There is a parameter to add annotations already. It is documented in the configuration section of the README.

Reply:
Won't a user who requires a more specific configuration look at the Configuration section when they need this?

It can indeed  also be shown in this example top make it immediately clear that this parameter is available.

Reply:
Users have asked for this and that means they use support / email / distract us.

If we put obvious things in an obvious place, we should need to spend less time re-explaining things.

That's how I see it.